static/app/components/modals/widgetViewerModal.tsx

```diff
@@ -463,8 +464,8 @@ const footerCss = css`
   margin: 0px -${space(4)} -${space(4)};
 `;
 
-const Container = styled('div')`
-  height: 300px;
+const Container = styled('div')<{height?: number | null}>`
+  height: ${p => (defined(p.height) ? `${p.height}px` : 'auto')};
```

Comment:
do we need to use the `defined` function here? 
```suggestion
  height: ${p => p.height ? `${p.height}px` : 'auto')};
```

Reply:
updated!